test/client.js

```diff
@@ -1,236 +1,323 @@
-      await wsClient.scaffold('example')('error')();
-    } catch (Error) {
-      test.errorCompare(Error, new Error('Return error'));
-    } finally {
+    'example/citiesByCountry': async (test) => {
+      const cities = await wsApi.example.citiesByCountry({
+        countryId: 1,
+      });
+      test.strictEqual(cities?.result, 'success');
+      test.strictEqual(Array.isArray(cities?.data), true);
       test.end();
-    }
-  });
-
-  metatests.testAsync('example/exception', async (test) => {
-
-    try {
-      await wsClient.scaffold('example')('exception')();
-    }
-    catch(Error){
-      test.errorCompare(Error, new Error('Example exception'));
-    }
-    finally {
+    },
+
+    'example/customError': async (test) => {
+      try {
+        await wsApi.example.customError();
+      } catch (customError) {
+        test.errorCompare(customError, new Error('Return custom error', 12345));
+        test.strictEqual(customError?.code, 12345);
+      } finally {
+        test.end();
+      }
+    },
+
+    'example/customException': async (test) => {
+      try {
+        await wsApi.example.customException();
+      } catch (customError) {
+        test.errorCompare(customError, new Error('Custom ecxeption', 12345));
+        test.strictEqual(customError?.code, 12345);
+      } finally {
+        test.end();
+      }
+    },
+
+    'example/error': async (test) => {
+      try {
+        await wsApi.example.error();
+      } catch (err) {
+        test.errorCompare(err, new Error('Return error'));
+      } finally {
+        test.end();
+      }
+    },
+
+    'example/exception': async (test) => {
+      try {
+        await wsApi.example.exception();
+      } catch (err) {
+        test.errorCompare(err, new Error('Example exception'));
+      } finally {
+        test.end();
+      }
+    },
+
+    'example/getClientInfo': async (test) => {
+      try {
+        const info = await wsApi.example.getClientInfo();
+        test.strictEqual(info?.result?.ip, '127.0.0.1');
+        test.strictEqual(info?.result?.token, wsToken);
+        test.strictEqual(info?.result?.accountId, '2');
+      } catch (err) {
+        console.log(err);
+      } finally {
+        test.end();
+      }
+    },
+
+    'example/redisSet + redisGet': async (test) => {
+      try {
+        const setting = await wsApi.example.redisSet({
+          key: 'MetarhiaExampleTest',
+          value: 1,
+        });
+        const getting = await wsApi.example.redisGet({
+          key: 'MetarhiaExampleTest',
+        });
+        test.strictEqual(setting?.result, 'OK');
+        test.strictEqual(getting?.result, '1');
+      } catch (err) {
+        test.errorCompare(err, new Error('Example exception'));
+      } finally {
+        test.end();
+      }
+    },
+
+    'example/resources': async (test) => {
+      try {
+        const resources = await wsApi.example.resources();
+        test.strictEqual(resources?.total, null);
+        // console.log({resources});
```

Comment:
Final version before landing should not contain commented console.log's

Reply:
ok, do I need to remove all commented code too?

Reply:
Just before landing, or you can add comment `// not for landing` if you added something not related to this PR or commented something just to make this code works but if it should be subject to separate PR